spec/models/serializers/work_queue/legacy_appeal_serializer_spec.rb

```diff
@@ -20,5 +20,47 @@
 
       expect(subject.serializable_hash[:data][:attributes][:appellant_address]).to eq serialized_appellant_address
     end
+
+    context "When all properties of the appellant_address are nil" do
+      let(:address) do
+        {
+          addrs_one_txt: nil,
+          addrs_two_txt: nil,
+          addrs_three_txt: nil,
+          city_nm: nil,
+          cntry_nm: nil,
+          postal_cd: nil,
+          zip_prefix_nbr: nil,
+          ptcpnt_addrs_type_nm: nil
+        }
+      end
+      let(:legacy_appeal) { create(:legacy_appeal, vacols_case: create(:case), veteran_address: address) }
+
+      it "Time zone is nil" do
+        expect(subject.serializable_hash[:data][:attributes][:appellant_address][:country]).to eq(nil)
+        expect(subject.serializable_hash[:data][:attributes][:appellant_tz]).to eq(nil)
+      end
+    end
+
+    context "When appellant_address country is nil" do
+      let(:address) do
+        {
+          addrs_one_txt: FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_1,
+          addrs_two_txt: FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_2,
+          addrs_three_txt: FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_3,
+          city_nm: FakeConstants.BGS_SERVICE.DEFAULT_CITY,
```

Comment:
These can be `nil` or even the following if you don't mind some randomness in this test:
```suggestion
          addrs_one_txt: [nil, FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_1].sample,
          addrs_two_txt: [nil, FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_2].sample,
          addrs_three_txt: [nil, FakeConstants.BGS_SERVICE.DEFAULT_ADDRESS_LINE_3].sample,
          city_nm: [nil, FakeConstants.BGS_SERVICE.DEFAULT_CITY].sample,
```